apps/worker/src/app/workflow/usecases/add-job/add-job.usecase.ts

```diff
@@ -169,8 +168,28 @@ export class AddJob {
       cronExpression?: string;
     } | null = null;
 
+    const subscriber = await this.subscriberRepository.findOne(
```

Comment:
I did a small refactor in this usecase to have a comon way of calculating next delay date from the cron